Samples/Qiskit/Driver.cs

```diff
@@ -22,14 +22,16 @@ static void Main(string[] args)
             if (apiKey.Contains("."))
             {
                 Console.Error.WriteLine("Did you put an api key in Driver.cs ? Without that, it will not work.");
-                return;
             }
-            var factory = new IbmQx4(apiKey); //Using different Factory
-            Console.WriteLine("Hadamard on IBMQx4");
-            for (int i = 0; i < 1; i++)
+            else
             {
-                var result = MeasurementOneQubit.Run(factory).Result;
-                Console.WriteLine($"Result of Hadamard is {result}");
+                var factory = new IbmQx4(apiKey); //Using different Factory
+                Console.WriteLine("Hadamard on IBMQx4");
+                for (int i = 0; i < 1; i++)
```

Comment:
?

Reply:
Moved the if statement, so it halts after the message. Otherwise one wouldn't see it.
The diff, makes it look wierd.